docs/docs/workflows/recovery.md

```diff
@@ -1,35 +1,34 @@
 # Recover your cluster
 
-Recovery of a Constellation cluster means getting a cluster back into a healthy state after too many concurrent node failures in the control plane.
+Recovery of a Constellation cluster means getting it back into a healthy state after too many concurrent node failures in the control plane.
 Reasons for an unhealthy cluster can vary from a power outage, or planned reboot, to migration of nodes and regions.
-Recovery events are rare, because Constellation is built for high availability and automatically and securely replaces failed nodes. When a node is replaced, Constellation's control plane first verifies the new node before it sends the node the cryptographic keys required to decrypt its [stateful disk](../architecture/images.md#stateful-disk).
+Recovery events are rare, because Constellation is built for high availability and automatically and securely replaces failed nodes. When a node is replaced, Constellation's control plane first verifies the new node before it sends the node the cryptographic keys required to decrypt its [state disk](../architecture/images.md#state-disk).
 
 Constellation provides a recovery mechanism for cases where the control plane has failed and is unable to replace nodes.
-The `constellation recover` command connects to a node, establishes a secure connection using [attested TLS](../architecture/attestation.md#attested-tls-atls), and provides that node with the key to decrypt its stateful disk and continue booting.
+The `constellation recover` command connects to a node, establishes a secure connection using [attested TLS](../architecture/attestation.md#attested-tls-atls), and provides that node with the key to decrypt its state disk and continue booting.
 This process has to be repeated until enough nodes are back running for establishing a [member quorum for etcd](https://etcd.io/docs/v3.5/faq/#what-is-failure-tolerance) and the Kubernetes state can be recovered.
```

Comment:
@daniel-weisse can you help me rephrase this paragraph to the new automated recovery?

Reply:
changed as suggested by Daniel